### ISSUE TITLE
Q: createCDPSession - will it be closed by page.close()?

### ISSUE THREAD

**Issue by hfisiteimprove** [puppeteer/puppeteer#7391]
Simple question on the createCDPSession.

Do we have to run a .detach() or will page.close() also close the CDPSession?
